### ISSUE TITLE
I can't install the package after create streamlinehq.json

### ISSUE THREAD

**Issue by lucasrusso95** [webalys-hq/streamlinehq-npm#24]
I got this error in terminal.

API error: connect ECONNREFUSED 127.0.0.1:8080
Error
Error: connect ECONNREFUSED 127.0.0.1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1146:16) {
  errno: -4078,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 8080
}

Someone can help me?

**Comment by mizziness** [comment]
@lucasrusso95  Try moving the info from the `streamline.json` file to an `.env` file?

Also having this issue, but intermittently.  I have my credentials set in `.env` but the kicker is that I use `yarn`.

```
error /somestring/tshipps/node_modules/@streamlinehq/streamlinehq: Command failed.
Exit code: 1
Command: npm start
Arguments:
Directory: /somestring/tshipps/node_modules/@streamlinehq/streamlinehq
Output:
npm WARN lifecycle The node binary used for scripts is /var/folders/1q/kmln0qwn5qx2qt9l6hlx4t806y_kfm/T/yarn--1621531573938-0.47448152348074335/node but npm is using /Users/tshipps/.nvm/versions/node/v12.13.0/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> @streamlinehq/streamlinehq@3.0.0 start /somestring/tshipps/node_modules/@streamlinehq/streamlinehq
> node build/index.js

Installing Streamline assets for streamline-regular, streamline-micro-bold families
API error: connect ECONNREFUSED 127.0.0.1:8080
Error
Error: connect ECONNREFUSED 127.0.0.1:8080
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1128:14) {
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 8080
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @streamlinehq/streamlinehq@3.0.0 start: `node build/index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @streamlinehq/streamlinehq@3.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I have tried passing through the `--scripts-prepend-node-path` flag with proper node location via `yarn add` but the same issue occurs.

It doesn't happen every time, though. All of a sudden, my build commands just start throwing errors for missing SVG files and checking does indeed show the `img` folder missing from `node_modules`. 

I can (sort of) work around it by installing it in a barebones folder without npm and copying over the missing `img` folder, or by temporarily using `npm` to install and then delete the resulting `.lock` file. But it's a really big pain in the butt :(

**Comment by lucasrusso95** [comment]
All times i get the same error

**Comment by mizziness** [comment]
@lucasrusso95 Hold on, it's `.streamlinehq.json`  I think, not `streamlinehq.json` - have you tried that?

**Comment by mizziness** [comment]
My workaround (new project, `npm`, copy folder) now no longer works, either.

```
$ npm install @streamlinehq/streamlinehq
npm ERR! code 1
npm ERR! path /somestring/node_modules/@streamlinehq/streamlinehq
npm ERR! command failed
npm ERR! command sh -c npm start
npm ERR! > @streamlinehq/streamlinehq@3.0.0 start
npm ERR! > node build/index.js
npm ERR!
npm ERR! Installing Streamline assets for streamline-regular, streamline-micro-bold families
npm ERR! Error
npm ERR! API error: connect ECONNREFUSED 127.0.0.1:8080
npm ERR! Error: connect ECONNREFUSED 127.0.0.1:8080
npm ERR!     at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1133:16) {
npm ERR!   errno: -61,
npm ERR!   code: 'ECONNREFUSED',
npm ERR!   syscall: 'connect',
npm ERR!   address: '127.0.0.1',
npm ERR!   port: 8080
npm ERR! }

npm ERR! A complete log of this run can be found in:
npm ERR!     /User/user/.npm/_logs/2021-05-20T18_52_21_642Z-debug.log
```

**Comment by lucasrusso95** [comment]
That's it @mizziness . I got the same error using in envs or file apart. Do you know how can i resolve it ?

**Comment by mizziness** [comment]
@lucasrusso95  Like I said, it's intermittent. Did you check in the debug log for the specific error?

**Comment by lucasrusso95** [comment]
158 error code ELIFECYCLE
159 error errno 1
160 error @streamlinehq/streamlinehq@3.0.0 postinstall: `npm start`
160 error Exit status 1
161 error Failed at the @streamlinehq/streamlinehq@3.0.0 postinstall script.
161 error This is probably not a problem with npm. There is likely additional logging output above.

**Comment by cbrwizard** [comment]
We're sorry for the inconvenience, the package has been published to version 3.0.0 before it was completely ready. 3.0.0 has been removed, please install it again with the latest released version - 2.1.0. I expect this issue to be solved there.

Here is the preview for 3.0 https://github.com/webalys-hq/streamlinehq-npm/pull/23 in case you're interested. We're planning a release next week.

**Comment by mizziness** [comment]
@cbrwizard Confirmed working after removal and reinstall <3 Thanks!
